tests.py

```diff
@@ -271,6 +271,23 @@ def test_sigfpe(self):
             3,
             'Floating point exception')
 
+    @skipIf(sys.platform != 'win32', 'specific to Windows')
+    def test_ignore_exception(self):
+        for exc_code in (
+            0xE06D7363,   # MSC exception ("Emsc")
+            0xE0434352,   # COM Callable Runtime exception ("ECCR")
+            0x40010006,   # Debug Print exception
```

Comment:
I'm ok with with this additional test, but I would prefer to first get it in Python upstream. Can you try to create a PR on Python to add this test? Please remove 0x40010006.

Reply:
If you don't want to create a PR on Python, I can do it. Tell me what you prefer.

Reply:
I can just remove it. I added it because it was the specific exception which was causing me issues, so I wanted to explicitly test on Windows that the case was fixed. I don't think there's any need to keep it.